src/main/java/Duke.java

```diff
@@ -1,10 +1,41 @@
+import java.util.ArrayList;
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Conversation.line();
+        Conversation.greeting();
+        Conversation.question();
+        Conversation.line();
+
+        boolean isContinue = true;
+        while(isContinue) {
+            String command = Conversation.readCommand();
+            String keyword = command.split(" ")[0];
```

Comment:
The variable name keyword is a little vague, perhaps change it to something that indicate what this variable holds?